src/run.sh

```diff
@@ -0,0 +1,3 @@
+make down
```

Comment:
This can be removed. Docker-compose automatically removes all the containers before restarting them. I don't know if make down creates problem initially when there are no containers started up

Reply:
I put it because when trying to `make` after that all container were up, I got errors (containers were not healthy). However if I `make down`, the `source loadenv.sh` and then `make` everything worked correctly. I don't know if this makes sense..

Reply:
Ok I will check later at home to check why the container were not working

Reply:
I checked and you are right. `make down` is not necessary. I will remove it